pilot/pkg/model/sidecar.go

```diff
@@ -236,6 +236,40 @@ type IstioEgressListenerWrapper struct {
 
 const defaultSidecar = "default-sidecar"
 
+// DefaultSidecarScopeForGateway builds a SidecarScope contains services and destinationRules for a given gateway/waypoint.
+func DefaultSidecarScopeForGateway(ps *PushContext, configNamespace string) *SidecarScope {
```

Comment:
Does this actually get called anywhere?

Reply:
Good, I forgot to commit the update locally